flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolInteractionsTest.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.flink.runtime.jobmaster.slotpool;
 
+import com.google.common.collect.Lists;
```

Comment:
You should follow the check style and take this import after flink.

Reply:
ok